.github/workflows/be-cd_dev-docker.yml

```
@@ -89,6 +89,9 @@ jobs:
         MONITORING_BINDING_PORT=${{ secrets.MONITORING_BINDING_PORT }}
         MONITORING_PORT=${{ secrets.MONITORING_PORT }}
         MONITORING_BASE_PATH=${{ secrets.MONITORING_BASE_PATH }}
+        
+        # Apply-form post URL generating format
+        APPLY_POST_BASE_URL=${{secrets.APPLY_POST_BASE_URL}}
```

```suggestion
        APPLY_POST_BASE_URL=${{ secrets.APPLY_POST_BASE_URL }}
```
뛰 어 쑤 기 부 탁 드 려 요 🙏🏻 🙏🏻 🙏🏻